src/commands/watch.json

```diff
@@ -18,6 +18,9 @@
         ],
         "key_specs": [
             {
+                "flags": [
```

Comment:
what should we do with this flag? (WATCH key key2)
And pfdebug? (should i skip it the check, or finish its arguments)

Reply:
i guess watch is basically the same as EXIST in terms of keyspec flags.. it allows you to know the key exists, or is modified, but doesn't "leak" the data.

regarding PFDEBUG, i suppose i told someone in the past not to bother.. but since it interferes with our correctness checking, maybe it's easier to just add the args rather than letting it stand out,